README.md

```diff
@@ -1,2 +1,55 @@
-# etcd
-A service for autodiscovery and configuration of applications running in containers
+# Autopilot pattern etcd
+
+*[Autopilot Pattern](http://autopilotpattern.io/) implementation of etcd*
+
+[![DockerPulls](https://img.shields.io/docker/pulls/autopilotpattern/etcd.svg)](https://registry.hub.docker.com/u/autopilotpattern/etcd/)
+[![DockerStars](https://img.shields.io/docker/stars/autopilotpattern/etcd.svg)](https://registry.hub.docker.com/u/autopilotpattern/etcd/)
+[![ImageLayers](https://badge.imagelayers.io/autopilotpattern/etcd:latest.svg)](https://imagelayers.io/?images=autopilotpattern/etcd:latest)
+[![Join the chat at https://gitter.im/autopilotpattern/general](https://badges.gitter.im/autopilotpattern/general.svg)](https://gitter.im/autopilotpattern/general)
+
+This repo is a demonstration of [etcd](https://coreos.com/etcd/docs/latest/) designed for self-operation according to the [Autopilot pattern](http://autopilotpattern.io/).
+
+An etcd cluster needs an external source of data for all the nodes to find each other initially. This can be a bootstrap service (another etcd cluster) or an SRV record. Triton CNS does not yet support SRV records, so until it does we're standing up a temporary single-node cluster to bootstrap the cluster. After the cluster is scaled-up we can remove the bootstrap node.
+
+
+### Getting started
+
+1. [Get a Joyent account](https://my.joyent.com/landing/signup/) and [add your SSH key](https://docs.joyent.com/public-cloud/getting-started).
+1. Install the [Docker Toolbox](https://docs.docker.com/installation/mac/) (including `docker` and `docker-compose`) on your laptop or other environment, as well as the [Joyent Triton CLI](https://www.joyent.com/blog/introducing-the-triton-command-line-tool) (`triton` replaces our old `sdc-*` CLI tools)
```

Comment:
Low priority: https://docs.joyent.com/public-cloud/api-access/cloudapi is a better choice for the Triton CLI install instructions